pages/game/roleScreen.tsx

```diff
@@ -8,16 +8,28 @@ interface RoleProps {
 }
 
 const RoleScreen: React.FC<RoleProps> = ({ data }) => {
-  const userId = useUserId();
 
   if (!data) {
     return (<></>);
   }
 
+  const unveilRole = ()=>{
+    const roleCard = document.querySelector(".roleCard")
```

Comment:
We definitely don't want to be mixing jquery into react components. Since we don't have the backend setup for revealing a role, we should take the revealing role out of this pr. 

In another pr we can do the revealing role logic. We don't want to load so much into one pr or else things get too complicated to review and to rollback if needed.

Reply:
Removed.